Packs/CommonScripts/Scripts/EmailAskUserResponse/EmailAskUserResponse.js

```diff
@@ -11,9 +18,6 @@ if (res[0].Type==entryTypes.note || res[0].Type==entryTypes.file) {
                 .replace(/__NL____NL__/g,'')
                 .replace(/__NL__/g,'\n');
     response = text.replace(/<(?:.|\n)*?>/gm, '').trim("\n").split("\n")[0].trim();
-    if (prefix && response.startsWith(prefix)) {
```

Comment:
moving it up doesn't change the logic a bit?

Reply:
It does, but this is actually an intended result, as removing it in the original place will remove the result of the white space trim. As such, this way achieves the intended behavior better